setup.py

```diff
@@ -7,6 +7,8 @@
     "lightgbm~=2.3",
     "xgboost~=0.90",
     "treelite_runtime~=2.2.2",
+    "scikit-learn~=1.0.2",
+    "pandas~=1.4.2"
```

Comment:
this risks to add overhead on memory of the pods - we should extra requires

Reply:
yeah it's quite heavy I think. Would be up to deployment to check the model type and install the extras, right?

Reply:
scikit-learn is already a dependency of lightgbm + I have removed pandas and now a "metadata.json" config file is required with the model file to get features names and order

Reply:
OK! looks like the simplest solution indeed